gse-copy-paste-afs/src/main/java/com/powsybl/gse/copy_paste/afs/exceptions/CopyDifferentFileSystemNameException.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.gse.copy_paste.afs.exceptions;
+
+import com.powsybl.gse.copy_paste.afs.CopyPasteException;
+
+/**
+ * @author Nassirou Nambiema <nassirou.nambiena at rte-france.com>
+ */
+public class CopyDifferentFileSystemNameException extends CopyPasteException {
+
+    public CopyDifferentFileSystemNameException(String msg) {
```

Comment:
Add default constructor with a predefined constant message so you may use it to match error from rest server.
like : 
```
public static String MESSAGE = "CopyPasteDifferentFileSystemNames";

public CopyDifferentFileSystemNameException() {
   super(MESSAGE);
}
```
and remove the parametrized constructor